schema.json

```diff
@@ -14,12 +14,36 @@
     "publisher": "Pulumi",
     "repository": "https://github.com/pulumi/pulumi-aws-static-website",
     "types": {
-        "aws-static-website:index:AtomicDeploymentArgs": {
+        "aws-static-website:index:CDNArgs": {
             "type": "object",
             "properties": {
-                "pulumiOrganization": {
-                    "type": "string",
-                    "description": "The Pulumi Organization you are deploying the website with. You only need to set this option if you are using an Organization in the Pulumi Service."
+                "forwardedValues": {
+                    "description": "The forwarded values configuration that specifies how CloudFront handles query strings, cookies and headers.",
+                    "$ref": "/aws/v5.28.0/schema.json#/types/aws:cloudfront/DistributionDefaultCacheBehaviorForwardedValues:DistributionDefaultCacheBehaviorForwardedValues"
+                },
+                "cloudfrontFunctions": {
```

Comment:
should we update this to be something like cloudfrontFunctionAssociations? it is probably kinda obvious though, so your call.